### ISSUE TITLE
Neo blocks are blank in preview view for Craft 3.7.4

### ISSUE THREAD

**Issue by ryguy2407** [spicywebau/craft-neo#483]
#### Description
When previewing an entry that has a Neo field in it I get a blank screen with no console errors. While the entry is still in draft mode the "view" link produces the same result. Once I save the entry the page behaves as expected with any new content added visible and working in preview mode as well. As soon as I add a new block the preview goes blank again until the page is saved.

Any pages that don't use a Neo field are working as expected.

#### Steps to reproduce
1. Open an existing entry
2. Hit preview (the layout is visible and working as expected in the preview pane)
3. Add a new block (All the neo blocks are blank where the content should be)
4. Add content to the added block (still the blocks are blank after the preview pane refreshes)
5. Close the preview and save the entry (everything works as expected from this point)

#### Other information
- Neo version: 2.11.1
- Craft version: 3.7.4
- Is eager-loading used? No.

Thanks :)

**Comment by texasbaptists** [comment]
I think I'm having the same issue. Here is some additional notes I was going to add before seeing an issue has been created with the same problem:

1. Preview works for drafts that were created before migrating to Craft 3.7 and Neo 2.11. If I make a new draft from an existing page the Neo Blocks aren't previewing. 
2. When I clone a block, the clone shows up in preview but the original still does not.
3. My setup has top level neo blocks of rows. Rows have column child blocks. Columns have children that are various types of blocks. The top level row blocks output html, but don't show child blocks. 
4. If I add `{{ block.children.count }}` to the template, it outputs `0` for where no preview is shown. If I clone the row that has children in it, the new copy will show the child blocks and the count will be accurate. The original row will remain showing `0`. 

tldr; 
I think it's actually child blocks that aren't showing up in live preview. At least for me.

**Comment by ttempleton** [comment]
Thanks for reporting this @ryguy2407, @texasbaptists, looks like the return of #451 (happening with revisions too)... will look into it.

**Comment by ttempleton** [comment]
Should be fixed now in Neo 2.11.2.

**Comment by texasbaptists** [comment]
@ttempleton Thanks! This update fixed it for me. Hope it fixes the issue for @ryguy2407 also.
